projects/common-components/src/lib/dialog/dialog.component.html

```diff
@@ -0,0 +1,30 @@
+<div class="dialog" (jsmuClickOutside)="closeDialog()">
```

Comment:
please use 'jsmu-' prefix for main blocks naming

Reply:
main blocks? do you mean class name for dialog block? or directive name

Reply:
yes

Reply:
done